main.py

```diff
@@ -178,13 +180,23 @@ def start_cron():
 
 def custom_data(post):
     return post.date_local >= datetime(2020, 3, 15)
+
+def zip_non_used_files():
+    for hashtag in HASHTAG_LABELS:
+        path = "./#" + hashtag
+        arqZip = zipfile.ZipFile("#" + hashtag + "ZIP.zip", "w", zipfile.ZIP_DEFLATED)
```

Comment:
💅 syntax sugar
```suggestion
        arqZip = zipfile.ZipFile(f"#{hashtag}ZIP.zip", "w", zipfile.ZIP_DEFLATED)
```

Reply:
The option `ZIP_DEFLATED` requires the `zlib` package to be installed in order to work properly (https://docs.python.org/3/library/zipfile.html). Why it is not included in the dependency requirements? 🤔

Reply:
Also, there is a reason why we preferred `zlib` compression instead of `bz2` or `lzma`? I don't have a strong argument for or against any of them, but I was just curious to hear your thoughts on this 😄

Reply:
Oh, I'll add the package in the code. Strange it worked without it, though.
And I decided for the ZIP_DEFLATED because it was stated to be the "numeric constant for the usual ZIP compression method". I thought it would be better to go for the "usual", but I admit I don't really know the differences of each compression method hahaha